Flea/Pages/Lopper.razor

```diff
@@ -0,0 +1,6 @@
+@page "/Loppe"

```

Comment:
File and `@page` name should be adjusted to match the internal naming convention (e.g. it should match the name of the model)

Reply:
Done